lib/index.js

```diff
@@ -676,15 +722,15 @@ exports.validate = internals.validate = function (email, options, callback) {
 
                             hyphenFlag = true;
                         }
-                            // Check if it's a neither a number nor a latin letter
-                        else if (charCode < 48 || charCode > 122 || (charCode > 57 && charCode < 65) || (charCode > 90 && charCode < 97)) {
+                            // Check if it's a neither a number nor a latin/unicode letter
+                        else if (charCode < 48 || (charCode > 122 && charCode < 192) || (charCode > 57 && charCode < 65) || (charCode > 90 && charCode < 97)) {
```

Comment:
@WesTyler I'm reading through some of this code as I refactor, and now I find myself asking where `192` came from. You wouldn't happen to recall, would you?

Reply:
Yeah, I believe 192 is the beginning of the Unicode "international" character set (À).

If I remember correctly, Unicode #s 123-191 are all non-character symbols.

Reply:
Ahh, ok. Thanks!